src/classic/clvm_tools/cmds.rs

```diff
@@ -730,28 +737,14 @@ pub fn launch_tool(stdout: &mut Stream, args: &[String], tool_name: &str, defaul
         .cloned()
         .unwrap_or_else(|| "*command*".to_string());
 
-    let dpr;
-    let run_program: Rc<dyn TRunProgram>;
     let mut search_paths = Vec::new();
-    match parsed_args.get("include") {
-        Some(ArgumentValue::ArgArray(v)) => {
-            let mut bare_paths = Vec::with_capacity(v.len());
-            for p in v {
-                if let ArgumentValue::ArgString(_, s) = p {
-                    bare_paths.push(s.to_string());
-                }
+    if let Some(ArgumentValue::ArgArray(v)) = parsed_args.get("include") {
+        let mut bare_paths = Vec::with_capacity(v.len());
+        for p in v {
+            if let ArgumentValue::ArgString(_, s) = p {
+                bare_paths.push(s.to_string());
             }
-            let special_runner =
-                run_program_for_search_paths(&reported_input_file, &bare_paths, extra_symbol_info);
-            search_paths = bare_paths;
-            dpr = special_runner.clone();
-            run_program = special_runner;
-        }
-        _ => {
-            let ordinary_runner =
-                run_program_for_search_paths(&reported_input_file, &Vec::new(), extra_symbol_info);
-            dpr = ordinary_runner.clone();
-            run_program = ordinary_runner;
+            search_paths = bare_paths.clone();
```

Comment:
When I was pushing through porting this I did actually make some mistakes in here since the structure of the code it started as was very different from what was natural in rust.  This simplifies it one time, but I'll go farther.